javalin-testtools/src/test/java/io/javalin/testtools/JavaTest.java

```diff
@@ -34,7 +36,7 @@ public void get_method_works() {
         JavalinTest.test((server, client) -> {
             server.get("/hello", ctx -> ctx.result("Hello, World!"));
             Response response = client.get("/hello");
-            assertThat(response.code()).isEqualTo(200);
+            assertThat(response.code()).isEqualTo(OK.getStatus());
```

Comment:
I'd personally add extension function to `Response` that returns `HttpCode` instance, so we don't have to call getStatus() each time in each class

Reply:
I don't have much Kotlin experience, I wouldn't know how to add an extension function. Can you help me with that?

Reply:
```kotlin
fun Response.javalinCode() = HttpCode.from(response.code()) // not sure about the enum method here
```

You can put this function "anywhere" and import it. It doesn't have to be inside a class (but it does have to be in a Kotlin file).

Reply:
I've appended
```kotlin
fun Response.httpCode() = HttpCode.forStatus(this.code)
```
to the JavalinTest.kt file. But it seems extension functions are just static to Java, such as:
```java
import static io.javalin.testtools.JavalinTestKt.httpCode;
...
assertThat(httpCode(response)).isEqualTo(OK);
```
Also, IDEA does not autocomplete them in Java files, only in Kotlin. 
Is the change woth it?

Reply:
To be honest, I thought @dzikoysk meant in Javalin's test suite, not in the `testtools` artifact.

I don't think this is necessary here.

Reply:
Just merge master, because I've already added this extension in the meantime:

* https://github.com/javalin/javalin/blob/037399917ac5aab33651dbebe7aaa91a7d539d09/javalin/src/test/java/io/javalin/testing/HttpUtil.kt#L38

Reply:
I've already merged master and updated some tests to use your method in the last commit.